checkov/common/bridgecrew/platform_integration.py

```diff
@@ -16,7 +16,9 @@
 
 UNAUTHORIZED_MESSAGE = 'User is not authorized to access this resource with an explicit deny'
 
-BC_API_URL = os.getenv('BC_API_URL',"https://www.bridgecrew.cloud/api/v1")
+BC_API_URL = os.getenv('BC_API_URL', "https://www.bridgecrew.cloud/api/v1")
+BC_SOURCE = os.getenv('BC_SOURCE', "cli")
+BC_REPO = os.getenv('BC_REPO', "")
```

Comment:
where do you use BC_REPO?

Reply:
I will use it in the next phase, from which repo of orbs/github action it arrived.

Reply:
is organisation part of the repo name? i.e. bridgecrewio/checkov ?

Reply:
Deleted